css/style.css

```
@@ -245,6 +245,95 @@ header .toggle {
   margin-top: 20px;
 }
 
+.section4 {
+  background-image: url(../img/section4-bg.jpg);
+}
+
+.section4 .cards {
+  display: flex;
+}
+
+.section4 .card {
+  width: 480px;
+  background-color: #ffffff;
+  box-shadow: 2px 0 2px 0 #cccccc;
+}
+
+.section4 .card + .card {
+  margin-left: 20px;
+}
+
+.section4 .cardImg {
+  position: relative;
+  color: #ffffff;
+}
+
+.section4 .cardImg::before {
+  content: '';
+  padding-top: calc(255 / 480 * 100%);
+  display: block;
+}
+
+.section4 .adultsAndStudents .backgroundImg{
```

ここのイメージはカードに使うイメージなので
```suggestion
.section4 .adultsAndStudents .cardImg{
```
でいいと思います。

---

> ここのイメージはカードに使うイメージなので
> 
> でいいと思います。

修正しました。
18b5acf5f1c6000755a920366a797d293afd0bee